provider/aws/processes.go

```diff
@@ -834,8 +848,14 @@ func (p *AWSProvider) generateTaskDefinition(app, process, release string) (*ecs
 		})
 	}
 
+	cd.Privileged = &s.Privileged
```

Comment:
Make this `cd.Privileged = aws.Boolean(s.Privileged)` rather than a direct reference to avoid possible side effects.

Reply:
@ddollar done